specs/PipelineDashboardAndActivitySecurityWithMultiplePermissions.spec

```diff
@@ -0,0 +1,62 @@
+Copyright 2018 ThoughtWorks, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+PipelineDashboardAndActivitySecurityWithMultiplePermissions
+===========================================================
+
+Setup of contexts
+* Permissions configuration - setup
+* Login as "admin" - setup
+* Using pipeline "P8, P9" - setup
+* With "1" live agents - setup
+* Capture go state "PipelineDashboardAndActivitySecurityWithMultiplePermissions" - setup
+
+PipelineDashboardAndActivitySecurityWithMultiplePermissions
+-----------------------------------------------------------
+
+tags: 6786, Permissions
```

Comment:
Lets add only relevent tags. Old tag values are of no use now since they refer to things in old mingle server and others